src/types.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * The libraries could return the data in the following formats.
+ */
+type RedisValue = string | number;
+
+/**
+ * The library sends Redis raw commands, so all we need to know are the
+ * 'raw-command-sending' functions for each redis client.
+ */
+export type SendCommandFn = (
+  ...args: RedisValue[]
```

Comment:
Some libraries like `ioredis` only allow an array of `string`s or `Buffer`s as arguments.

Reply:
Changed it to `string[]` in c71b74b